pom.xml

```diff
@@ -66,7 +66,7 @@
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.version>3.9.9</maven.version>
 
-    <flink.version>1.14.6</flink.version>
+    <flink.version>1.16.3</flink.version>
```

Comment:
Is it better to use 1.20 version?

Reply:
@SteNicholas Okay, I have updated to version 1.20.0, PTAL